jwst/outlier_detection/outlier_detection.py

```diff
@@ -145,7 +160,7 @@ def do_detection(self):
         if pars['resample_data'] is True:
```

Comment:
ACK! You're going to get a PEP8 evil eye from @pllim on this!

Reply:
`flake8` is the one who does the actual complaining. Don't shoot the messenger! :sweat_smile:

Reply:
Time to improve my code compliance... ;-)